pageObject_model/CalculatorHomePage.js

```diff
@@ -0,0 +1,178 @@
+import { elementToBeClickable } from 'wdio-wait-for';
+import { GoogleCloudHomePage } from './GoogleCloudHomePage.js';
+
+export class CalculatorHomePage extends GoogleCloudHomePage {
+    constructor() {
+        super();
+        this.loc = '//md-option/*[contains(text(), "Paid: Ubuntu Pro")]';
+        this.machineClassLocatorItem =
+            '//*[@id="select_container_93"]/md-select-menu//md-option/*[contains(text(), "RAW")]';
+        this.datacenterLocatorItem =
+            '//md-select-menu[@class="md-overflow"]//*[contains(text(),"RAW")]';
+        this.commitedUsageLocatorItem =
+            '//*[@id="select_container_115"]/md-select-menu//md-option/*[contains(text(), "RAW")]';
+        this.numberGrusLocatorItem =
+            '//*[@id="select_container_454"]/md-select-menu//md-option/*[contains(text(), "RAW")]';
+        this.itemXpathXXX = '//md-option/*[contains(text(), "RAW")]';
+        this.urlCalculatorHome = 'https://cloud.google.com/products/calculator';
+        this.frameOneXpath = '//*[@id="cloud-site"]/devsite-iframe/iframe';
+        this.frameTwoXpath = '//*[@id="myFrame"]';
+        this.checkedModeXpath = './/div[@title="Compute Engine"]';
+        this.formComputeEngineXpath = '//form[@name="ComputeEngineForm"]';
+        this.inputNumberInstancesXpath = '//input[@name="quantity"]';
+        this.itemXpath = '//*[@id="select_option_91"]/div[1]';
+        this.selectOperatingSystemXpath =
+            '//*[@ng-model="listingCtrl.computeServer.os"]/*[@class="md-select-value"]//div';
+        this.operatingSystemListXpath = '//*[@id="select_container_89"]';
+        this.machineClassSelectXpath =
+            '//*[@ng-model="listingCtrl.computeServer.class"]/*[@class="md-select-value"]//div';
+        this.machineClassListXpath = '//*[@id="select_container_93"]';
+        this.machineClassItemXpath = '//*[@id="select_option_90"]/div';
+        this.seriesSelectXpath =
+            '//*[@ng-model="listingCtrl.computeServer.series"]/*[@class="md-select-value"]//div';
+        this.seriesListXpath = '//*[@id="select_container_101"]';
+        this.machineTypeSelectXpath =
+            '//*[@ng-model="listingCtrl.computeServer.instance"]/*[@class="md-select-value"]//div';
+        this.machineTypeListXpath = '//*[@id="select_container_103"]';
+        this.checkboxXpath =
+            '//*[@aria-label="Add GPUs"]/div[@class="md-container md-ink-ripple"]';
+        this.numberGrusSelectXpath =
+            '//*[@placeholder="Number of GPUs"]/*[@class="md-select-value"]//div';
+        this.numberGrusListXpath = '//*[@id="select_container_454"]';
+        this.numberGrusItemXpath =
+            '//*[@id="select_option_462"]/div[@class="md-text ng-binding"]';
+        this.GruTypeSelectXpath = '//*[@aria-label="GPU type"]';
+        this.GruTypeListXpath = '//*[@value="NVIDIA_TESLA_K80"]/ancestor::div';
```

Comment:
1) Typo: gpu
2) Only classes should start with capitals.

Reply:
done